.github/actions/file-diff/index.js

```diff
@@ -56,8 +56,8 @@ async function run() {
 		// --------------- End evaluation  ---------------
 
 		/** Split the data by component package */
-		const COMPONENTS = splitDataByComponent(pathOutput, diffOutput);
```

Comment:
@pfulton Needed to update this to understand where the assets live - it was hardcoded to look for them in components and tokens and ui-icons don't live there. I also updated this to capture the "main" value from the package instead of assuming we want to compare the index.css file.